docs/ops/deployment/native_kubernetes.md

```diff
@@ -116,10 +116,12 @@ $ kubectl port-forward service/<ServiceName> 8081
 - `NodePort`: Exposes the service on each Node’s IP at a static port (the `NodePort`). `<NodeIP>:<NodePort>` could be used to contact the Job Manager Service. `NodeIP` could be easily replaced with Kubernetes ApiServer address.
 You could find it in your kube config file.
 
-- `LoadBalancer`: Default value, exposes the service externally using a cloud provider’s load balancer.
+- `LoadBalancer`: exposes the service externally using a cloud provider’s load balancer.
 Since the cloud provider and Kubernetes needs some time to prepare the load balancer, you may get a `NodePort` JobManager Web Interface in the client log.
 You can use `kubectl get services/<ClusterId>` to get EXTERNAL-IP and then construct the load balancer JobManager Web Interface manually `http://<EXTERNAL-IP>:8081`.
 
+  <span class="label label-warning">Warning!</span> Your JobManager (which can run arbitary jar files) might be exposed to the public internet, without authentication.
```

Comment:
Is this a problem of Flink, or the provider?

Reply:
This Flink config is just forwarded to Kubernetes, and the purpose o this setting is to expose the service publicly via a loadbalancer: https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types
I used the word *might* because I can imagine that in some setups, the LB only opens the service to a VPN. 
But when I experimented with Flink on K8s on Google Cloud, the JobManager was public to the internet by default.
That's why I added this warning here.